services/movie_session.py

```diff
@@ -42,3 +42,10 @@ def update_movie_session(
 
 def delete_movie_session_by_id(session_id: int) -> None:
     MovieSession.objects.get(id=session_id).delete()
```

Comment:
Consider adding error handling for the `delete_movie_session_by_id` function to manage cases where a movie session with the given ID does not exist. This can prevent the function from raising a `DoesNotExist` exception.